types/core.d.ts

```diff
@@ -16,6 +16,10 @@ export interface TaskTracker {
     [key: number]: TaskInfo;
 }
 
+export interface Region {
+    [key: string]: string;
```

Comment:
What do the key and string value represent? Is the value a CSS selector?

Reply:
Yes; I can add comment to explain.